rel/files/riak-cs-debug

```diff
@@ -487,23 +555,44 @@ fi
 
 if [ 1 -eq $get_cfgs ]; then
     mkdir_or_die "${start_dir}"/"${debug_dir}"/config/.info
-    cd "${start_dir}"/"${debug_dir}"/config
 
-    # Use dump to execute the copy. This will provide a progress dot and
-    # capture any error messages.
-    dump cp_cfg cp -R "$cs_etc_dir"/*.conf* .
+    # Capture the needed files from the `cs_etc_dir` directory.
+    cd $cs_etc_dir
 
-    # If the copy succeeded, then the output will be empty and it is unneeded.
-    if [ 0 -eq $? ]; then
-        rm -f cp_cfg
+    # Convert the list of file blobs to a list of `"! -name <blob> `.
+    # For example if `RIAK_EXCLUDE="*.key mySecretFile"`, this will set
+    # `exclude="! -name *.key ! -name mySecretFile"`.
+    exclude=""
+    for i in `echo $RIAK_EXCLUDE`; do
+        exclude="${exclude}! -name $i "
+    done
+    if [ 0 -eq $get_ssl_certs ]; then
+        for i in `echo "'*.pfx' '*.p12' '*.csr' '*.sst' '*.sto' '*.stl' '*.pem' '*.key'"`; do
+            exclude="${exclude}! -name $i "
+        done
     fi
 
-    mkdir_or_die "${start_dir}"/"${debug_dir}"/generated.config/.info
-    cd "${start_dir}"/"${debug_dir}"/generated.config
+    # Compose the `find` command that will exclude the above list of files,
+    # being aware that an empty `\( \)` will cause a failure in the `find`.
+    if [ -n $exclude ]; then
```

Comment:
It seems `$exclude` needs double-quote for avoiding error when $exclude is empty.